packages/fogbender-vue/src/components/FogbenderConfig.ts

```diff
@@ -0,0 +1,48 @@
+import { Env, Token } from "fogbender";
+import { defineComponent, inject, onMounted, onUnmounted, PropType, watch } from "vue-demi";
+import { addVersion, fogbender } from "../util";
+export default defineComponent({
+  name: "FogbenderConfig",
+  props: {
+    clientUrl: {
+      type: String,
+      required: false,
+    },
+    env: {
+      type: Object as PropType<Env>,
```

Comment:
isn't that a String?

Reply:
Yes this is of type string its the syntax of vue to give custom prop types in vue

![image](https://user-images.githubusercontent.com/47587853/187772631-85daccab-c0e6-4422-88d6-f94ef90de56c.png)

If you will see in the example it is expecting evn of type Env not env of type object

Reply:
but `type: String as PropType<Env>,` also works

Reply:
Acknowledged

Reply:
Done